### ISSUE TITLE
[SF][1.2.1][Switch][A11y] Aria label of Switch does not be announced by screen reader

### ISSUE THREAD

**Issue by i323808** [SAP/ui5-webcomponents#5151]
### **Bug Description**
Aria label of Switch does not be announced by screen reader.

### **Expected Behavior**
The label associated with the Switch component should be announced by screen reader.

### **Steps to Reproduce**
1. Go to https://codesandbox.io/s/switch-accessiblename-hezxew?file=/src/App.js
2. Focus the Switch
3. JAWS does not announce the label information.

### **Isolated Example**
https://codesandbox.io/s/switch-accessiblename-hezxew?file=/src/App.js

### **Context**
- UI5 Web Components version: 1.2.1
- OS/Platform: Windows10
- Browser: Chrome
- Affected component: Switch
- Screen reader: JAWS2020

### **Log Output / Stack Trace / Screenshots**
<img width="993" alt="switch_ariaLabel" src="https://user-images.githubusercontent.com/30744017/165893249-9fe11c2d-52f1-409a-a235-320b4a38ba72.png">

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: {...}
- Business impact: {...}

**Comment by niyap** [comment]
Hello @SAP/ui5-webcomponents-topic-b,

Could you please look over the issue? 
The "change" event is not fired when the value is empty. I tried in the components test page and I managed to reproduce the issue with small difference. When I select a range from the picker and move the focus somewhere else, then access the ui5-daterange-picker again, select the value with Ctrl + A and delete it, on focus out the "change" event is fired but after that on each next time that I repeat the mentioned steps it is not fired anymore with an empty value.
Please check the isolated samples provided from the reporter. The issue is reproducible from the first time as well there.

Thank you in advance!

Kind Regards,
Niya

**Comment by unazko** [comment]
Hello @i323808,

Could you please retest via the provided sample: https://codesandbox.io/s/switch-accessiblename-hezxew?file=/src/App.js
It is not reproducible at my side any longer. When I first tested it was reproducible though.

Best Regards,
Boyan Rakilovski

**Comment by i323808** [comment]
Hi @unazko ,

I retested it. Aria label of Switch is working fine with ui-webcomponents 1.4.0 and ui5-webcomponents-react 0.24.0. 

Thanks,
Jane

**Comment by unazko** [comment]
Hi @i323808,

As it is not a common practice to transport fixes in older releases, I'm closing the issue.

Best Regards,
Boyan

**Comment by vladitasev** [comment]
This seems to have been fixed by https://github.com/SAP/ui5-webcomponents/pull/4927
